docs/userGuide/siteJsonFile.md

```diff
@@ -81,6 +81,10 @@ Here is a typical `site.json` file:
 
 <include src="deployingTheSite.md#warning-about-baseUrl" />
 
+<box type="warning">
+
+Note: `baseUrl` does not support live reload as it should not change in `markbind serve` mode.
```

Comment:
Let's update the definition here as well with this https://deploy-preview-1529--markbind-master.netlify.app/userguide/glossary

Minor wording nit:
```suggestion
Note: `baseUrl` does not support live preview as there is no use case for changing it in during `markbind serve`.
```

Otherwise, Lgtm! 👍 

Let's wait and see if @raysonkoh has anything to add